tests/unit/conftest.py

```diff
@@ -16,14 +16,18 @@
     "good": {
         "steps": {
             "pvs_like_case_study": {
-                "implementation": "pvs_like_python",
+                "implementation": {
```

Comment:
In my head, this change was going to be optional. But I think this is a UI thing we can do for now and reassess later.